src/state/keyed.rs

```diff
@@ -134,6 +134,11 @@ where
         }
     }
 
+    /// Returns the number of non-empty keys present in the map.
+    pub fn len(&self) -> usize {
+        self.map_reader.len()
+    }
```

Comment:
Oh, wait - turns out the `clippy` failure wasn't from the master branch at all:

> error: item `state::keyed::KeyedRateLimiter<K, A, C>` has a public `len` method but no corresponding `is_empty` method

Can you implement that one as well?

Reply:
Yup!